src/main/java/org/folio/service/finance/transaction/EncumbranceService.java

```diff
@@ -124,7 +124,7 @@ public Future<Void> createEncumbrances(List<EncumbranceRelationsHolder> relation
   }
 
 
-  public Future<Void> updateEncumbrancesOrderStatus(CompositePurchaseOrder compPo, RequestContext requestContext) {
+  public Future<Void> updateEncumbrancesOrderStatus(CompositePurchaseOrder compPo, RequestContext requestContext) {    logger.info("STEP {}", "createPOandPOLines");
```

Comment:
We can update log message and add order id to this log

Reply:
done